src/keygen.ts

```diff
@@ -0,0 +1,21 @@
+import { flags } from './deps.ts'
+import { revokeNet } from './utils.ts'
+import { EDWARDS25519SHA512BATCH, keygen as cryptoKeygen, serializeKey } from './lib/crypto.ts'
+
+export const keygen = async (args: string[]) => {
+  await revokeNet()
+  const parsedArgs = flags.parse(args)
+  const key = cryptoKeygen(EDWARDS25519SHA512BATCH)
+  const textEncoder = new TextEncoder()
+  const result = textEncoder.encode(JSON.stringify({
+    version: '1.0.0',
+    pubkey: serializeKey(key, false),
+    privkey: serializeKey(key, true)
+  }))
+
+  if (parsedArgs['out']) {
+    await Deno.writeFile(parsedArgs['out'], result)
+  } else {
+    await Deno.stdout.write(result)
+  }
```

Comment:
Request:

- output 2 files, one for the public key (for verifying with `chel latestState`), and one for the private key for signing with `chel manifest` (like SSH)
- remove the STDOUT. you can have a default filename based on the key type (make sure to output to STDOUT where this file is saved, and check to see if it already exists, in which case, just add a suffix like `id_edssa.1.pub` and `id_edssa.2.priv`) or something to that effect